### PR TITLE
fix breadcrumbs collapsed a11y failure

### PR DESCRIPTION
**Pull request by bvandrc**
collapsed button should be tabbable and have `role="button"`

### CODE REVIEW

File: packages/core/src/components/breadcrumbs/breadcrumbs.tsx [hunk 1]
```diff
@@ -97,7 +97,7 @@ export class Breadcrumbs extends AbstractPureComponent<BreadcrumbsProps> {
             <OverflowList
                 collapseFrom={collapseFrom}
                 minVisibleItems={minVisibleItems}
-                tagName="ul"
+                tagName="ol"
```
Comment (ggdouglas) [inline packages/core/src/components/breadcrumbs/breadcrumbs.tsx]: Is there a reason we are changing the default element type here from an unordered to an ordered list element? Is it possible to separate this change from the other a11y failure?

File: packages/core/src/components/breadcrumbs/breadcrumbs.tsx [hunk 2]
```diff
@@ -127,7 +127,12 @@ export class Breadcrumbs extends AbstractPureComponent<BreadcrumbsProps> {
                     content={<Menu>{orderedItems.map(this.renderOverflowBreadcrumb)}</Menu>}
                     {...popoverProps}
                 >
-                    <span className={Classes.BREADCRUMBS_COLLAPSED} />
+                    <span
+                        aria-label="collapsed breadcrumbs"
```
Comment (ggdouglas) [inline packages/core/src/components/breadcrumbs/breadcrumbs.tsx:136]: While adding an `aria-label` is a great accessibility improvement, we should avoid hardcoding the string "collapsed breadcrumbs" directly in the component. This approach doesn't support localization for international users.

Instead, can we introduce a new prop, something like `expandText`, to allow consumers of this component to provide a localized or customized string? We can set the default value of that prop to a string that best describes the action of this button.

```suggestion
                        aria-label={this.props.expandText}
```

File: packages/core/src/components/breadcrumbs/breadcrumbs.tsx (hunk 1 above)
Comment (bvandrc) [reply packages/core/src/components/breadcrumbs/breadcrumbs.tsx]: Breadcrumbs _are_ ordered so they should be tagged as such.

Here is the aria example: https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/examples/breadcrumb/

Comment (bvandrc) [reply packages/core/src/components/breadcrumbs/breadcrumbs.tsx]: > Is it possible to separate this change

https://github.com/palantir/blueprint/pull/7007

File: packages/core/src/components/breadcrumbs/breadcrumbs.tsx (hunk 2 above)
Comment (bvandrc) [reply packages/core/src/components/breadcrumbs/breadcrumbs.tsx:136]: Are you down for a _default_ of `"collapsed breadcrumbs"`? Or do you want no `aria-label` at all?

Comment (bvandrc) [reply packages/core/src/components/breadcrumbs/breadcrumbs.tsx:136]: Can set via `overflowButtonProps`: https://github.com/palantir/blueprint/pull/6985/commits/85519823178f115b902d4572a1cbbfbd43658ac8

Comment (ggdouglas) [reply packages/core/src/components/breadcrumbs/breadcrumbs.tsx:136]: > Are you down for a _default_ of `"collapsed breadcrumbs"`? Or do you want no `aria-label` at all?

Having `"collapsed breadcrumbs"` as the default seems fine to me.

**Review by ggdouglas — APPROVED** [review]

**Pull request merged**